packages/browser/src/browser/index.ts

```diff
@@ -320,6 +321,11 @@ async function loadAnalytics(
   // this is an ugly side-effect, but it's for the benefits of the plugins that get their cdn via getCDN()
   if (settings.cdnURL) setGlobalCDNUrl(settings.cdnURL)
 
+  if (options.initialPageview) {
+    // capture the page context early, so it's always up-to-date
+    preInitBuffer.push(new PreInitMethodCall('page', []))
```

Comment:
This doesn't store all the page context like the snippet does...does it? My understanding is that this would queue a page call that will be invoked after some async work has been completed (which gives the address of the page time to change)

Reply:
Yes, it stores the page context!

Reply:
Ah right, forgot `preInitBuffer` isn't a regular array.